src/Data/Char/Unicode/Internal.purs

```diff
@@ -105,17 +105,20 @@ instance showCharBlock :: Show CharBlock where
                          <> " }"
 
 
-gencatZP :: Int
-gencatZP = 67108864
+gencatPF :: Int
```

Comment:
Not certain why the changes are here for these codes. I generated this file as explained in the README. If it needs to be changed back to the previous order, then something must be different about the unicodedata file right?